values.yaml

```diff
@@ -103,6 +103,10 @@ global:
   ssl:
     enabled: false
     mode: "prefer"
+  
+  connectionLimit:
+    enabled: false
+    limit: 5
```

Comment:
As I mentioned above, we can prob pull this out to be part of the `astronomer` subchart values.yaml